### PR TITLE
Added syncd SAI dump to sysdump script

### PR DESCRIPTION
**Pull request by andrii-savka**
It should merged only after following request:
https://github.com/Azure/sonic-sairedis/pull/199

### CODE REVIEW

File: scripts/generate_dump [hunk 1]
```diff
@@ -285,6 +285,11 @@ main() {
     save_redis "2" "COUNTERS_DB"
 
     save_cmd "docker exec -it syncd saidump" "saidump"
+    
+    local sai_dump_filename="/tmp/syncd_sai_dump"
```
Comment (stcheng) [inline scripts/generate_dump]: add timestamp?

Comment (lguohan) [inline scripts/generate_dump]: should be only on mellanox platform since it is the only platform that supports this.

**Comment by andrii-savka** [comment]
Fixed above remarks. 
Thanks.

**Comment by andrii-savka** [comment]
The saisdkdump tool is already merged. 
All remarks are fixed. 
So we can merge this request.

**Comment by lguohan** [comment]
cannot merge this one, since the change is checked into 1.0.3 branch. It can only be merged after 1.0.3 branch is merged into master.

**Pull request merged**